src/cursor/find_cursor.ts

```diff
@@ -72,7 +72,8 @@ export class FindCursor<TSchema = any> extends ExplainableCursor<TSchema> {
     const options = {
       ...this.findOptions, // NOTE: order matters here, we may need to refine this
       ...this.cursorOptions,
-      session
+      session,
+      signal: this.signal
```

Comment:
```suggestion
```

I don't feel strongly - this isn't necessary because the `FindOptions` holds the signal (that's how Aggregate works now actually).  I'd either always add the signal (align AggregationCursor with FindCursor) or always rely on the options (remove this.signal here).

Reply:
I'd say we should at least type the `findOptions` property as `FindOptions & Abortable`

Reply:
Can we update the options types of the constructor too?  Looks like findOptions is now `FindOptions & Abortable` but the constructor still only takes `FindOptions`.